src/fidesops/service/connectors/query_config.py

```diff
@@ -255,6 +255,13 @@ def format_clause_for_query(
         """Returns clauses in a format they can be added into SQL queries."""
         return f"{field_name} {operator} :{field_name}"
 
+    def format_clause_for_query_in(
+        self, field_name: str, query_data_names: List[str]
+    ) -> str:
+        """Returns clauses in a format they can be added into SQL queries."""
+        in_clause = ", ".join(query_data_names)
+        return f"{field_name} IN ({in_clause})"
+
```

Comment:
This method will currently live on every `SQLQueryConfig`, even where we don't need special `IN` clause formatting, which would be confusing for any developer checking the class def.

To stick with convention, could we instead:
- create a new `MicrosoftSQLServerQueryConfig(SQLQueryConfig)`  class
- override `format_clause_for_query()` on that class that returns the correctly formatted clause where `operator="IN"`
- add `query_config()` to the `MicrosoftSQLServerConnector` class and have it return the new `MicrosoftSQLServerQueryConfig`